Makefile

```diff
@@ -5,7 +5,7 @@ SHELL           = /bin/bash
 
 IMAGE_REGISTRY 	?= ghcr.io
 IMAGE_REPO     	?= kedacore
-VERSION 		?= main
+VERSION 		?= canary
```

Comment:
?

Reply:
This is because in this repo the `main` tag, but maybe I can set it during externally, you are right